### ISSUE TITLE
How to keep in sync with CRAN's settings?

### ISSUE THREAD

**Issue by eddelbuettel** [wch/r-debug#10]
(Not sure this is a valid issue myself so feel free to close)

I once again got a love note from CRAN about UBSAN, and as has happened in the past cannot reproduce. It is a wee bit maddening that they expect us to jump over hoops, but don't really share their rope.   Do you have any ideas / plans / suggestions / ... about we could do that better?

(My current case in point is RcppStreams which passes your `RDsan` and `RDcsan` yet gets  [BDR's clang-san to yell at it](https://www.stats.ox.ac.uk/pub/bdr/memtests/clang-ASAN/RcppStreams/00check.log) over some issue deep down in Boost.)

**Comment by eddelbuettel** [comment]
In related news: how do you build to not hit this?

```sh
edd@rob:~/git/r-devel-san-clang(master)$ docker build --tag localsan:latest  --security-opt seccomp=unconfined .                                                                                                  
Error response from daemon: The daemon on this platform does not support setting security options on build                                                                                                        
edd@rob:~/git/r-devel-san-clang(master)$
```

**Comment by eddelbuettel** [comment]
FWIW I updated mine (still at https://github.com/rocker-org/r-devel-san-clang) and can reproduce with it.

It would still be nice if we had a way to keep up with CRAN :-/

**Comment by wch** [comment]
I recently encountered a similar situation where CRAN was getting significant warnings with R-devel and gcc 8, but this docker image had an older compiler so it wasn't immediately easy to reproduce. I agree that it would be nice to have a simple way to keep up with CRAN but it's not clear to me how.

**Comment by eddelbuettel** [comment]
We could try to pool resources.  Given non communicative "some of them" are, we may have to bite the bullet and build on fedora core (eeek).  Come to think about it, maybe not.  "Some of them" are clearly obsessive-compulsive and follow re-releases of compilers at times which we may not have time for.  (Then again, there is an apt-get'able archive with pre-release compilers for IIRC gcc and clang too).

Maybe start with an IFTT monitor of the Oxford page?   What saved my bacon this time was that @krlmlr  had done such fine work on my Dockerfile that I already had the special flags to ignore errors at buildtime -- so I "just" had to update to clang 7.  With that the clang sanitizer reproduced on my package what they were howling about and a "fixed" package is now on CRAN.

Sometimes I have hope to pin one them down and start a conversation about Dockerizing their resources.  But then most often reality sets back ...

So in short count me in "not clear to me how".  Makes two of us.
